spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -1034,7 +1061,7 @@ public void testUnpartitionedPartitionsTable() {
   @Test
   public void testPartitionsTable() {
     TableIdentifier tableIdentifier = TableIdentifier.of("db", "partitions_test");
-    Table table = createTable(tableIdentifier, SCHEMA, PartitionSpec.builderFor(SCHEMA).identity("id").build());
+    Table table = createTable(tableIdentifier, SCHEMA, SPEC);
```

Comment:
Are these changes needed? I don't have a problem with them, but I don't think they are required right?

Reply:
No, just a clean up I had in the original PR, I can split this into another pull request if needed